scripts/subset_hail_table.py

```diff
@@ -67,14 +66,23 @@ def main(
     if biallelic:
         ht = ht.filter(hl.len(ht.alleles) == 2)
 
-    # create the output path; make sure we're only ever writing to test
-    actual_output_path = output_path(output_root).replace(
-        f'cpg-{get_config()["workflow"]["dataset"]}-main',
-        f'cpg-{get_config()["workflow"]["dataset"]}-test',
+    # create the output path; only ever writing to test
+    output_path = dataset_path(
+        os.path.join(get_config()['workflow']['output_prefix'], output_root),
+        access_level='test',
     )
 
     # write the Table to a new output path
-    ht.write(f'{actual_output_path}.ht', overwrite=True)
+    if out_format in ['ht', 'both']:
+        table_path = f'{output_path}.ht'
+        ht.write(table_path, overwrite=True)
+        logging.info(f'Wrote new table to {table_path!r}')
```

Comment:
Do you need the `!r` here and below in the f-string? Isn't `table_path` just a simple string?

Reply:
AFAIK there was an update to pylint so it rejects format strings surrounded by quotes in favour of using !r. There wasn’t really any need for quotes here, just something to make the path written clearer in the logs.